commit 8af2caf7307d8002d88c377c58eba8cae75a2109
Author: Breno Leitao <leitao@debian.org>
Date:   Mon Jul 29 01:52:11 2024 -0700

    failcmd: make failcmd.sh executable
    
    Change the file permissions of tools/testing/fault-injection/failcmd.sh to
    allow execution.  This ensures the script can be run directly without
    explicitly invoking a shell.
    
    Link: https://lkml.kernel.org/r/20240729085215.3403417-1-leitao@debian.org
    Signed-off-by: Breno Leitao <leitao@debian.org>
    Reviewed-by: Akinobu Mita <akinobu.mita@gmail.com>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>

diff --git a/tools/testing/fault-injection/failcmd.sh b/tools/testing/fault-injection/failcmd.sh
old mode 100644
new mode 100755